README.md

```diff
@@ -96,56 +108,118 @@ An example Jenkins file [SonarQube-StaticScan-Jenkinsfile](./jenkins/SonarQube-S
 
 Additional details regarding the `openshift/jenkins-slave-zap` image can be found here; [OWASP ZAP Security Vulnerability Scanning](https://github.com/BCDevOps/openshift-components/tree/master/cicd/jenkins-slave-zap)
 
 ### Quality Badges
+
 Now that you are scanning your code you can publish the summary of the results using badges in your project's top-level ReadMe file.
 
-For SonarQube versions <7.1 you will need to use the [SVG Badges](https://github.com/QualInsight/qualinsight-plugins-sonarqube-badges) plug-in, which is bundled in the `bcgovimages/sonarqube:6.7.5` image.  Starting with SonarQube 7.1, badges are available from the platform without a plugin.
+For SonarQube versions <7.1 you will need to use the [SVG Badges](https://github.com/QualInsight/qualinsight-plugins-sonarqube-badges) plug-in, which is bundled in the `bcgovimages/sonarqube:6.7.5` image. Starting with SonarQube 7.1, badges are available from the platform without a plugin.
 
 # Examples
 
 [TheOrgBook](https://github.com/bcgov/TheOrgBook) provides an example of the concepts outlined here, and demonstrates the use of static code scanning, ZAP report integration, and quality badges.
 
+# SonarCloud
+
+Many teams in bcgov have started to adopt [SonarCloud](https://sonarcloud.io/) and have moved away from using SonarQube on OpenShift.
+
+SonarCloud is a cloud service offered by SonarSource and based on SonarQube. SonarQube is a widely adopted open source platform to inspect continuously the quality of source code and detect bugs, vulnerabilities and code smells in more than 20 different languages.
+
+Key benefits of SonarCloud are:
+
+- Enhance your workflow with continuous code quality scanning
+- It automatically analyzes and decorates pull requests on GitHub
+- It can be invoked from your workstation, OpenShift pipeline and your GitHub actions on top of that it will run for every PR automatically
+- It will save precious OpenShift resources
+- It is free for Open Source projects (that's us!)
+
+## Steps to start using SonarCloud
+
+1. Identify a person in your team, who will be the admin for your project in SonarCloud
+2. Make sure that that person is a member of bcgov in github
+3. Sign up for SonarCloud with your github Account
+4. Add a `.sonarcloud.properties` file to your project root
+
+**Example:**
+
+```
+sonar.projectName=ppr
+
+# Path to sources
+sonar.sources=ppr-api/src,ppr-ui/src
+#sonar.exclusions=
+#sonar.inclusions=
+
+# Path to tests
+sonar.tests=ppr-api/tests,ppr-ui/tests/unit
+#sonar.test.exclusions=
+#sonar.test.inclusions=
+
+# Source encoding
+sonar.sourceEncoding=UTF-8
+
+# Exclusions for copy-paste detection
+#sonar.cpd.exclusions=
+```
+
+5. Ask the Developer Experience group to add your project (bcgov is the owner of your project and Dev.Experience manages the bcgov account )
```

Comment:
New request type for sonarcloud integration is available: https://github.com/BCDevOps/devops-requests